deployments.md

```diff
@@ -0,0 +1,155 @@
+
+Deployed by NTT deployment owner:
+- an ntt token
+Deployed by wormhole:
+- wormhole core layer
+- wormhole relayer (you can use wormhole's public one or deploy your own)
+- specialized relayer (you can use wormhole's public one or deploy your own)
+
+### High level description of the process:
+To have a multi-chain NTT setup working a few things need to happen.
+- Deploy managers on EVM
+- Register managers on NTT Token
+- Deploy Transceivers on EVM
+- Deploy NTT Program on Solana
+- Set Mint authority to NTT program
+- Initialize NTT program on Solana
+- Cross register managers (let each manager know about the other managers)
+- Cross register transceivers
+
+## EVM Deployment:
+
+### Environments & Configuration:
+These scripts are implemented in way that helps keep track of the deployed addresses by reading contract addresses from json files that can be commited to the repository. Multiple environments are supported through the use of the `ENV` environment variable.
+To create or use a new env all that needs to be done is to create a new directory on `evm/ts-scripts/config/<my-env>` with the name of the env you want to use and then set the environment variable ENV to the same value (ie on linux: `export ENV="my-env"`)
+Inside the configuration directory, there needs to be a few files that will contain all the configuration you need to set up a full NTT deployment:
+- `chains.json` contains the configuration used by the deployment runner such as what rpcs will be used or what chains will it run scripts against
+- `contracts.json` this file contains the addresses of all contracts related to this deployment and will be used any time a script needs to know about the address of other of the contracts. Addresses of contracts will be written to this file as the deployment progress (ie after deploying managers we'll add the manager addresses here)
+- `managers.json` contains the configuration that will be applied to managers
+- `transceivers.json` contains the configuration that will be applied to transceivers
+- `peers.json` contains the configuration that will be used for contract cross-registrations
+
+> When a script deploys new contracts, it outputs the resulting addresseses on the directory `ts-scripts/output/<scripts-name>/<timestamp>.json (aside from logging them). Whenever we need to configure the addresses of a contract that we have just deployed we'll fetch the new addresses from those files.
+
+### Deployment Steps:
+- create/update `evm/config/<env>/contracts.json` as needed
+- create/update `evm/config/<env>/managers.json` as needed
+- create/update `evm/config/<env>/transceivers.json` as needed
+- Set environment variables:
+```
+export FOUNDRY_PROFILE=prod
+export WALLET_KEY="ledger"
+export LEDGER_BIP32_PATH=""
+export ENV=testnet
+```
+- Build:
+```shell
+cd evm
+
+forge build # builds contracts to evm/out
```

Comment:
This is the !prod build right? should it be with the `make build-evm-prod` at the root?

Reply:
I didn't notice that one. I'll update to use that one instead.